sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteDataConfiguration.java

```diff
@@ -1,13 +1,18 @@
 // SPDX-License-Identifier: MIT
 package com.mercedesbenz.sechub.commons.model;
 
+import java.util.Optional;
+
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class SecHubRemoteDataConfiguration {
 
     public static final String PROPERTY_TYPE = "type";
     public static final String PROPERTY_LOCATION = "location";
+    public static final String PROPERTY_CREDENTIALS = "credentials";
+
+    private SecHubRemoteCredentialConfiguration credentials;
 
     private String type;
 
```

Comment:
Because this is not supported by system test framework at the moment, I created 
- #3069
as a follow up issue.